src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -413,16 +414,16 @@ transactionSign (
             return e;
     }
 
-    if (!tx_json.isMember ("Sequence"))
-        tx_json["Sequence"] = ledgerFacade.getSeq ();
-
     if (!tx_json.isMember ("Flags"))
         tx_json["Flags"] = tfFullyCanonicalSig;
 
     if (verify)
     {
+        if (!tx_json.isMember ("Sequence"))
```

Comment:
Just a note that if Nik fixes all these quoted strings (and there are lots), then I'm gonna have a heckuva rebase of TransactionSign in my future.  The m-of-n branch does major surgery on TransactionSign.

Fixing the quoted stings would be the right thing to do, but it might want to be a separate commit.